### ISSUE TITLE
Grid tile names seem reversed from dep-grid

### ISSUE THREAD

**Issue by jessjaco** [digitalearthpacific/dep-tools#62]
`odc.geo.gridspec.GridSpec` produces an index in `(x, y)` order, as stated here: https://odc-geo.readthedocs.io/en/latest/_api/odc.geo.gridspec.GridSpec.tiles.html#odc.geo.gridspec.GridSpec.tiles. This corresponds to `(column, row)` order.

But the working grid we've been using for some things has `tile_id` in `(row, column)` order (see https://github.com/digitalearthpacific/dep-grid/blob/main/grid_pacific.geojson). I've used that in e.g. coastlines.

We need to be consistent. I suggest we maintain `(column, row)` order as defined by the gridspec.

**Comment by alexgleith** [comment]
>We need to be consistent. I suggest we maintain (column, row) order as defined by the gridspec.

Agree.

**Comment by jessjaco** [comment]
It looks like the original tile_id column is in fact correct (it is in column, row order), the issue is that the names for the row and column columns should be switched (row is column and column is row)

**Comment by alexgleith** [comment]
Ok, so all the data I wrote is fine?

FYI, when DEA write data they do `x10` and `y05` rather than just `10/05`... which we probably should do too. Removes ambiguity.

Example: `https://data.dea.ga.gov.au/?prefix=derivative/ga_ls8c_nbart_gm_cyear_3/3-0-0/x11/y18/2014--P1Y/`

**Comment by jessjaco** [comment]
Yeah I did a quick check of the geomad stuff, everything looks consistent.

If you want to change the directory naming that's fine with me , let's just do it ASAP.

**Comment by alexgleith** [comment]
Do you think it's a good idea? I don't really mind, so long as we're consistent as you said.
